frontend/src/components/Rating.js

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+
+const Rating = (props) => {
+  const { rating } = props
+  return (
+    <div className="rating">
+      <span>
+        <i
+          className={
+            rating >= 1
+              ? "fa fa-star"
+              : rating >= 0.5
+              ? "fa fa-star-half-o"
+              : "fa fa-star-o"
+          }
```

Comment:
It's a bit too complex operations to have directly in JSX - it would be better to move it to separate function